src/features/stock-ticker/redux/actions.ts

```diff
@@ -0,0 +1,18 @@
+import { UPDATE_STOCK_TICKER } from "./constants";
+import { ActionWithPayload } from "../../../store/actions";
+export type StockTickerData = {
+  latestPrice: number;
+  latestUpdate: Date;
```

Comment:
q: Isn't this `Date` a string sent from the server?

Reply:
Yes, it sends the `Date` from the server, you can console.log it and have alook